GoodMorning/GoodMorning/TODOHeaderView.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  TODOHeaderView.swift
+//  GoodMorning
+//
+//  Created by Miro on 2023/08/04.
+//
+
+import UIKit
+
+class TODOHeaderView: UICollectionReusableView {
+
+    weak var delegate: TODOHeaderViewDelegate?
+
+    private lazy var editButton: UIButton = {
+        let button = UIButton()
+        button.addTarget(
+            self,
+            action: #selector(editButtonTapped),
+            for: .touchUpInside
+        )
+        button.setTitle("Edit", for: .normal)
+        button.titleLabel?.font = .systemFont(ofSize: 15)
+        button.setTitleColor(.darkGray, for: .normal)
+
+        return button
+    }()
+
+    private lazy var plusTODOButton: UIButton = {
+        let button = UIButton()
+        button.addTarget(
+            self,
+            action: #selector(plusButtonTapped),
+            for: .touchUpInside
+        )
+        let SFSymbolConfiguration = UIImage.SymbolConfiguration(
+            pointSize: 20,
+            weight: .bold,
+            scale: .large
+        )
+        let SFSymbolImage = UIImage(
+            systemName: "plus.circle",
+            withConfiguration: SFSymbolConfiguration
+        )
+
+        button.setImage(
+            SFSymbolImage?.withTintColor(.black, renderingMode: .alwaysOriginal),
+            for: .normal
+        )
+
+        return button
+    }()
```

Comment:
이 버튼은 아침 루틴을 추가하는 버튼으로, + 이미지를 갖고 있어요.

해당 버튼을 UIKit에 내장되어있는 UIButton.type을 이용해보는건 어떨까요?

```suggestion
 private lazy var plusTODOButton: UIButton = {
        let button = UIButton(type: .contactAdd)
        button.tintColor = .black
        button.addTarget(
            self,
            action: #selector(plusButtonTapped),
            for: .touchUpInside
        )
        
        return button
    }()

```

Reply:
이러한 방법이 있었군요! 훨씬 좋네요😄

> 커밋: ef1fa86